sphinx_gallery/gen_gallery.py

```diff
@@ -674,6 +674,7 @@ def generate_gallery_rst(app):
 
             recommender_params = copy.deepcopy(gallery_conf["recommender"])
             recommender_params.pop("enable")
+            recommender_params.pop("rubric_header", None)
```

Comment:
Good catch! Would we be able to get a test to check config processing?

Reply:
Is something similar as done in [c69f075](https://github.com/sphinx-gallery/sphinx-gallery/pull/1125/commits/c69f0752cde6d5723599d342d42c67cb778cf346) what you had in mind?

Reply:
Yes thanks